wallet/src/ui/app/components/sui-apps/filters.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import cl from 'classnames';
+import { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { NavLink } from 'react-router-dom';
+
+import st from './Filters.module.scss';
+
+const ELEMENT_ID = '#sui-apps-filters';
+function activeTagsFilter({ isActive }: { isActive: boolean }) {
+    return cl({ [st.active]: isActive }, st.filter);
+}
+
+function AppFiltersPortal() {
+    const [ready, setReady] = useState(false);
+    const content = document.querySelector(ELEMENT_ID) as HTMLElement;
+
+    const filterTags = [
+        {
+            name: 'Playground',
+            link: 'apps',
+        },
+        {
+            name: 'Active Connections',
+            link: 'apps/connected',
+        },
+    ];
+
+    useEffect(() => {
```

Comment:
This really should be a mutation observer, because in the event that this DOM node actually is missing and then appears, this won't be triggered again.